docs/onboard-to-seed.md

```diff
@@ -0,0 +1,50 @@
+# Onboard to SEED
+
+Currently Public Officers can onboard their device to SEED by logging into the TechPass portal using non-SE GSIB machine.
+
+<div class="warn">
+ <ul>
+ <li>SEED is applicable only for <b>non-GSIB device</b>.</li>
+ <li>If you are a vendor or contractor who needs to be onboarded to SEED, contact your project manager or reporting officer.</li>
+ <li>Click <a href="https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices">here</a> for more information about SEED.</li>
+ </ul>
+</div>
+
+_To onboard your device to SEED:_
+
+1. Go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
+
+<kbd>![log in](assets/images/onboarding/po-non-se/log-in-with-techpass.png ':size=90%')</kbd>
+
+2. Choose your TechPass account.
+
+<kbd>![choose-account](assets/images/onboarding/po-non-se/choose-account.png)</kbd>
+
+?> If you are a public officer and not logged into your WOG account, you will be prompted to log in to it first before proceeding to TechPass account.
+
+3. You will be prompted to approve your TechPass login. Approve it on your Authenticator app.
+
+<kbd>![approve-signin](assets/images/onboarding/po-non-se/approve-sign-in.png)</kbd>
+
+4. Hover over your account name and click **My Account**.
+
+<kbd>![view-account](assets/images/onboarding/po-non-se/view-account-or-profile.png)</kbd>
+
+5. Under **SEED Status**, either click **Request for SEED** _(this will show up if you have not been assigned SEED license)_
```

Comment:
Generally, we follow British English in Singapore. So it should be Licence when it is used as a noun.

Should the option be "Request for SEED Licence" instead of "Request for SEED".